blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,86 @@
+
+From here, you can follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows. Dev box will not have access to your local files, so you will need to redownload any source code that you need to get started developing. When you are finished with your development session and have shut down the virtual machine in the Dev Box portal, your machine's state will be saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-MicrosoftDevBox.jpg)
+
+## Takeaways about Microsoft Dev Box
+
+Microsoft Dev Box is a great tool when your team is ready for a larger level of investment in the Windows platform. Its feature offerings of cloud-based infrastructure, pre-configured dev boxes, and Intune secured workstations are great for enterprise-scale development.
+
+Microsoft Dev Box does require several admin and configuration steps to get started, so if a developer is looking to quickly experiment with RNW within a VM, Parallels or another virtual machine offering may be a good option to get started with. Then, once the developer is ready to scale their Windows VM usage across many developers to build large production experiences, Microsoft Dev Box will become a better option.
+
+## Overall Takeaways
+
+In this post, I looked at two options for building React Native for Windows apps from Mac devices using Windows VMs. As I close out my content for today, I want to emphasize a few key points. Below is a table which captures the scenario comparisons between Microsoft Dev Box and Parallels.
+
+| Feature                                          | Microsoft Dev Box | Parallels |
+| ------------------------------------------------ | ----------------- | --------- |
+| Locally installed virtual machine software       |                   | YES       |
+| Cloud-based virtual machine                      | YES               |           |
+| Easy to follow documentation                     | YES               | YES       |
+| Great for quick experimentation with Windows     |                   | YES       |
+| Great for enterprise development                 | YES               |           |
+| Scale Windows VM usage across several developers | YES               |           |
+| Pre-configured dev boxes                         | YES               |           |
+
+As you reach the end of reading, I encourage you to give React Native for Windows and Windows VMs a try! To get started with React Native for Windows visit [aka.ms/reactnative](https://microsoft.github.io/react-native-windows/). If you have any feedback or questions for the React Native for Windows team, you can reach out to us on GitHub at [microsoft/react-native-windows](https://github.com/microsoft/react-native-windows). We want to hear about your developer experience!
```

Comment:
I like the "give it a try" framing; I think we should add a note that if your company is going to invest in supporting Windows, having dedicated Windows hardware is the way to go.